### ISSUE TITLE
Shouldn't babel-runtime be in dependencies rather than devDependencies?

### ISSUE THREAD

**Issue by shaimo** [shakacode/react_on_rails#320]
It is being used in ReactOnRails.js:
```
var _assign = require('babel-runtime/core-js/object/assign');
```
And would yield an error if babel-runtime is not installed...

**Comment by justin808** [comment]
Can you give me the line number, @shaimo?

**Comment by justin808** [comment]
BTW, the dependencies for Babel have been changing. The best is to refer to: 

https://github.com/shakacode/react-webpack-rails-tutorial/

**Comment by shaimo** [comment]
@justin808 I see it in line 7 of my 'node_modules/react-on-rails/node_package/lib/ReactOnRails.js' with
```
└── react-on-rails@3.0.6 
```

**Comment by justin808** [comment]
Thanks @shaimo.

Any chance that you can submit a PR for the needed changes?

**Comment by kalashnikovisme** [comment]
have the same problem after getting started 

```javascript
ReactOnRails.js?0003:7 Uncaught Error: Cannot find module "babel-runtime/core-js/object/assign"
```

**Comment by justin808** [comment]
Yes. I'll get this merged shortly.

https://github.com/shakacode/react_on_rails/pull/361/files

**Comment by justin808** [comment]
Given:
https://babeljs.io/docs/plugins/transform-runtime/

And given that I have not seen this issue, it would be helpful to know why we need this.

I don't have babel-runtime here: https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/package.json#L51

@alexfedoseev do you know about this one?

**Comment by alexfedoseev** [comment]
@justin808 Well, in package's `.babelrc` there is [`transform-runtime`](https://github.com/shakacode/react_on_rails/blob/master/node_package/.babelrc#L3). It means that the compiled lib is using `babel-runtime` (not bundling it, but linking to it). So to make compiled lib work, there must be `babel-runtime` installed as `dependency` of the package (not `devDependency`).

> And given that I have not seen this issue
Two guesses why it's happening: 
1. you might have `babel-runtime` installed globally, so webpack can resolve it from there, rather than from project's local `node_modules`.  
2. Or, since npm 3.x tree is flat, some other package depends on `babel-runtime`, so it available in `react-webpack-rails-tutorial`'s `node_modules`.

Bottom line: `babel-runtime` should be added as `dependency` to make clean install work.

**Comment by justin808** [comment]
@alexfedoseev Maybe this should be a peer dependency as well?
